lib/views/after_auth_screens/events/explore_events.dart

```diff
@@ -138,31 +138,34 @@ class ExploreEvents extends StatelessWidget {
                                 ),
                                 SizedBox(
                                     height: SizeConfig.screenHeight! * 0.027),
-                                ListView.builder(
-                                    physics:
-                                        const NeverScrollableScrollPhysics(),
-                                    shrinkWrap: true,
-                                    itemCount: model.events.length,
-                                    itemBuilder:
-                                        (BuildContext context, int index) {
-                                      return GestureDetector(
-                                        key: index == 0
-                                            ? homeModel?.keySECard
-                                            : null,
-                                        onTap: () {
-                                          navigationService.pushScreen(
-                                              "/eventInfo",
-                                              arguments: {
-                                                "event": model.events[index],
-                                                "exploreEventViewModel": model
-                                              });
-                                        },
-                                        child: EventCard(
-                                          event: model.events[index],
-                                          isSearchItem: false,
-                                        ),
-                                      );
-                                    }),
+                                Padding(
```

Comment:
Please remove the padding from here. This will reduce user experience. We should give users as much screen area as possible. Padding should be added to the end card. As the user reaches the bottom of the screen (after scrolling) the card there should have padding space to let the user see the data which is hidden by the floating button.